docs/MANUAL_CLOUD_TESTS.md

```diff
@@ -0,0 +1,36 @@
+# Ubuntu SRU: Manual cloud test design
```

Comment:
> Thanks for this, it's much clearer what you're aiming for here! And, of course, I have questions. ;-)
> 
> We've talked in the past about converting the templates to be scripts that we can call, capture the output of, and be done; it seems to me like this PR is most of the way to that already. Do we want to change the language we're using from "template" to "script"?

I think I get what you are looking for. WDYT about [this commitish  renaming sru-templates->sru-scripts](https://github.com/cloud-init/ubuntu-sru/pull/113/commits/f865a88654136cb881c5a658ea8c722d4d9bd6fd)

> 
> I suppose that this would mean that we couldn't bundle specific bug testing into the single manual run. In my mind, this is an acceptable trade-off, because I rarely if ever do that, but perhaps other people's mileage varies there.

WDYT about [this commitish which adds a function that can optionally use run-parts on a supplemental-tests directory](https://github.com/cloud-init/ubuntu-sru/pull/113/commits/f1d2b382834f7b47b30c528613a427f5be5e5a92).

Reply:
> I think I get what you are looking for. WDYT about this commitish renaming sru-templates->sru-scripts

I like it for Azure, but I suggest that we retain `sru-templates` to contain the unmigrated (and therefore still) templates and move them to `sru-scripts` as they are updated for the new pattern.

> WDYT about this commitish which adds a function that can optionally use run-parts on a supplemental-tests directory.

I'm less sure about this; in the interests of unblocking the SRU process, can we drop it from this PR and then consider it as a follow-up?

Reply:
@OddBloke thanks. Yes to both.